package.json

```diff
@@ -125,11 +126,12 @@
     "docs": "webpack --config .config/webpack.config.js",
     "eslint": "clear && eslint ./ && echo \"No errors found.\"",
     "eslint:watch": "esw -w --clear --cache",
+    "precommit": "yarn eslint && yarn test && yarn dist && yarn bundle:check && git add dist && git add docs",
```

Comment:
Do not add file without explicit approval from user and remove eslit, test and bundle:check commands.